compiler/luci/pass/src/CircleShapeInferencepass.test.cpp

```diff
@@ -0,0 +1,354 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/CircleShapeInferencePass.h"
+
+#include <loco.h>
+
+#include <luci/IR/CircleNodes.h>
+
+#include <gtest/gtest.h>
+
+/**
+ * This test is for checking when original shape of node should be changed first.
```

Comment:
does `original shape of node` ? mean `shape of input to a node` ? or is it `shape of a node before update` ?

Reply:
`shape of a node before update` is what I intended.
I will change the whole sentence to represent more specific goal of the test.